src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpMethodRepoDAO.scala

```diff
@@ -37,9 +47,19 @@ class HttpMethodRepoDAO( methodRepoServiceURL: String )( implicit system: ActorS
 
   private def when500( throwable: Throwable ): Boolean = {
     throwable match {
-      case ure: spray.client.UnsuccessfulResponseException => ure.responseStatus.intValue/100 == 5
-      case ure: spray.httpx.UnsuccessfulResponseException => ure.response.status.intValue/100 == 5
+      case ure: spray.client.UnsuccessfulResponseException => ure.responseStatus.intValue / 100 == 5
```

Comment:
I think it's a mistake to remove the httpx case.  Which you'll receive depends on whether the exception is thrown by sendReceive or unmarshal.